terraform/features.go

```diff
@@ -0,0 +1,17 @@
+package terraform
+
+import (
+	"fmt"
+	"os"
+)
+
+// This file holds feature flags for the next release
+
+var featureOutputErrors bool
+
+func init() {
+	featureOutputErrors = os.Getenv("TF_OUTPUT_ERRORS") != ""
+	if featureOutputErrors {
+		fmt.Println("WARNING: output errors feature enabled")
```

Comment:
I remember we've had problems before with early output interfering with... something. Perhaps it was the plugin RPC protocol? If things seem to be working okay then this is probably not a big deal since this is off by default anyway, but just wanted to flag it since I can't remember what exactly the problem was when we did something like this before.

Reply:
Yeah, I wanted this to be kind of loud, but I didn't think about it getting built into a provider.
This is internal only anyway, so it's probably better to drop it for now.